.github/workflows/APIDigesterCheck.yml

```diff
@@ -0,0 +1,42 @@
+name: Swift API Digester Functionality Check
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # This will run the action every night at midnight
+  workflow_dispatch:  # Allows manual triggering
+
+jobs:
+  check-swift-api-digester:
+    runs-on: macos-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
+      with:
+        fetch-depth: 0
```

Comment:
Do we really need to fetch all history for all branches and tags?

Reply:
Changed! Thanks!